docs/releases.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Releases
+keywords: apache, hudi, release, data lake, upsert,
+sidebar: home_sidebar
+permalink: releases.html
+toc: true
+summary: "Apache Hudi (incubating) Releases Page"
+---
+
+
+## Latest Stable Release
+  * Stable Release : `0.5.0-incubating`
```

Comment:
can you grep for "Latest Version" (as it show up above side nav on the site) and also update that to .. 
"Latest: 0.5.0-incubating"